lab-evan-erick-michael/model/bitmap-constructor.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var BitmapConstructor = module.exports = function(type, size, pixelStart, dBISize, colorPalette) {
+  this.type = type;
+  this.size = size;
+  this.pixelStart = pixelStart;
+  this.dBISize = dBISize;
+  this.colorPalette = colorPalette;
+  this.grayPalette = [];
+  this.randomPalette = [];
+  this.invertedPalette = [];
+  this.scaledPalette = [];
+  this.transformedGrayPalette = [];
+  this.transformedRandomPalette = [];
+  this.transformedPaletteInverted = [];
+  this.transformedScaledPalette = [];
+};
```

Comment:
@kcirekcom @jmpaik @EWPS07 `bitmap-contstructor` looks good!  It could be useful to also save the raw buffer as a property on the bitmap constructor as well (ex: `this.bitmap = data`)